src/System.ServiceModel.Federation/src/System/ServiceModel/Federation/WSTrustUtilities.cs

```diff
@@ -0,0 +1,163 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics.Tracing;
+using System.IdentityModel.Tokens;
+using System.ServiceModel.Security;
+using System.ServiceModel.Security.Tokens;
+using Microsoft.IdentityModel.Protocols;
+using Microsoft.IdentityModel.Protocols.WsSecurity;
+using Microsoft.IdentityModel.Protocols.WsTrust;
+using SecurityToken = System.IdentityModel.Tokens.SecurityToken;
+
+namespace System.ServiceModel.Federation
+{
+    /// <summary>
+    /// <see cref="WSTrustChannelSecurityTokenProvider"/> has been designed to work with the <see cref="WSFederationHttpBinding"/> to send a WsTrust message to obtain a SecurityToken from an STS. The SecurityToken is
+    /// added as an IssuedToken on the outbound WCF message.
+    /// </summary>
+    public static class WSTrustUtilities
+    {
```

Comment:
All members of this class are internal. I think you intended for the class to be internal too?

Reply:
addressed in: ab9b3c1cb00f558dcd6ac6f3b02663ace5e25750